app/jobs/socure_docv_results_job.rb

```diff
@@ -38,6 +38,7 @@ def create_analytics(
   def socure_document_verification_result
     DocAuth::Socure::Requests::DocvResultRequest.new(
       document_capture_session_uuid:,
+      analytics: @analytics,
     ).fetch
```

Comment:
```
      timer = JobHelpers::Timer.new
      response = timer.time('vendor_request') do
        DocAuth::Socure::Requests::DocvResultRequest.new(document_capture_session_uuid:).fetch
      end
      ....
      analytics.idv_socure_verification_data_requested(
        ...
        vendor_request_time_in_ms: timer.results['vendor_request'],
        ...
      )
      response
```

this seems like it may be the right place to generate the analytics.  you could use the DocvResultResponse extras to access data like state, dob_year, state_id_type, ...etc.  Check out [this example](https://github.com/18F/identity-idp/blob/74f0ff081fb96eb02bb4f2665a01c7648f114f11/app/forms/idv/api_image_upload_form.rb#L93) for a similar scenario